admin/test/utils.go

```diff
@@ -26,16 +26,17 @@ import (
 	"testing"
 	"time"
 
-	v3statusgrpc "github.com/envoyproxy/go-control-plane/envoy/service/status/v3"
-	v3statuspb "github.com/envoyproxy/go-control-plane/envoy/service/status/v3"
 	"github.com/google/uuid"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/admin"
-	channelzpb "google.golang.org/grpc/channelz/grpc_channelz_v1"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials/insecure"
-	"google.golang.org/grpc/internal/xds"
+	"google.golang.org/grpc/internal/testutils/xds/bootstrap"
 	"google.golang.org/grpc/status"
+
+	v3statusgrpc "github.com/envoyproxy/go-control-plane/envoy/service/status/v3"
+	v3statuspb "github.com/envoyproxy/go-control-plane/envoy/service/status/v3"
+	channelzpb "google.golang.org/grpc/channelz/grpc_channelz_v1"
```

Comment:
This is fine, but do we have grouping style conventions on pbs around the codebase? I've seen these declared in arbitrary orderings (the groups), and sometimes their intermingled.

Reply:
We don't have a strict convention on this. But I find it nicer when the proto imports are grouped separately, especially for our case since we have to import some of these packages two times (pb/grpc).

See: go/go-style/decisions#import-grouping